coalib/collecting/Collectors.py

```diff
@@ -82,7 +82,9 @@ def collect_files(file_paths, log_printer, ignored_file_paths=None,
         collected_files, file_globs_with_files = [], []
 
     _warn_if_unused_glob(log_printer, file_paths, file_globs_with_files,
-                         "No files matching '{}' were found.")
+                         "No files matching '{}' were found. "
+                         'If this rule is not required, you can remove it '
+                         'from your .coafile to deactivate this warning.')
```

Comment:
I have gone ahead and implemented it in this fashion i.e. the section that includes the missing glob is named in the warning.
Changes were quite minor, thanks to optional parameters, but in hindsight i should probably have moved that change into its own commit (in case i have to revert it).

Looking forward to feedback.